.circleci/config.yml

```
@@ -0,0 +1,18 @@
+---
+version: 2
+jobs:
+  build:
+    machine:
+      image: ubuntu-2004:202101-01
```

same as looker-spoke-default
```suggestion
  deploy:
    docker:
      - image: cimg/base:2020.01
```